lib/include/zig.h

```diff
@@ -143,1457 +157,1347 @@
 #define memory_order_release 3
 #define memory_order_acq_rel 4
 #define memory_order_seq_cst 5
+#define zig_atomic(type) type
 #define zig_cmpxchg_strong(obj, expected, desired, succ, fail) zig_unimplemented()
-#define zig_cmpxchg_weak  (obj, expected, desired, succ, fail) zig_unimplemented()
+#define   zig_cmpxchg_weak(obj, expected, desired, succ, fail) zig_unimplemented()
 #define zig_atomicrmw_xchg(obj, arg, order) zig_unimplemented()
-#define zig_atomicrmw_add (obj, arg, order) zig_unimplemented()
-#define zig_atomicrmw_sub (obj, arg, order) zig_unimplemented()
-#define zig_atomicrmw_or  (obj, arg, order) zig_unimplemented()
-#define zig_atomicrmw_xor (obj, arg, order) zig_unimplemented()
-#define zig_atomicrmw_and (obj, arg, order) zig_unimplemented()
+#define  zig_atomicrmw_add(obj, arg, order) zig_unimplemented()
+#define  zig_atomicrmw_sub(obj, arg, order) zig_unimplemented()
+#define   zig_atomicrmw_or(obj, arg, order) zig_unimplemented()
+#define  zig_atomicrmw_xor(obj, arg, order) zig_unimplemented()
+#define  zig_atomicrmw_and(obj, arg, order) zig_unimplemented()
 #define zig_atomicrmw_nand(obj, arg, order) zig_unimplemented()
-#define zig_atomicrmw_min (obj, arg, order) zig_unimplemented()
-#define zig_atomicrmw_max (obj, arg, order) zig_unimplemented()
-#define zig_atomic_store  (obj, arg, order) zig_unimplemented()
-#define zig_atomic_load   (obj,      order) zig_unimplemented()
+#define  zig_atomicrmw_min(obj, arg, order) zig_unimplemented()
+#define  zig_atomicrmw_max(obj, arg, order) zig_unimplemented()
+#define   zig_atomic_store(obj, arg, order) zig_unimplemented()
+#define    zig_atomic_load(obj,      order) zig_unimplemented()
 #define zig_fence(order) zig_unimplemented()
 #endif
 
-#include <stdint.h>
-#include <stddef.h>
-#include <limits.h>
+#if __STDC_VERSION__ >= 201112L
+#define zig_noreturn _Noreturn void
+#define zig_threadlocal thread_local
```

Comment:
I see `#define zig_threadlocal` near the top of the file too

Reply:
Good catch,  looks like I got sidetracked in the middle of changing this.